doc/source/operations/upgrading.rst

```diff
@@ -66,8 +66,46 @@ Some things to watch out for:
   required by the secure RBAC policies.
 * Application credentials generated before the existence of any implicit roles
   will not be granted those roles. This may include the ``reader`` role, which
-  is referenced in some of the new secure RBAC policies.  See `Keystone bug
-  2030061 <https://bugs.launchpad.net/keystone/+bug/2030061>`_.
+  is referenced in some of the new secure RBAC policies. This issue has been
+  seen in app creds generated in the Yoga release. See `Keystone bug 2030061
+  <https://bugs.launchpad.net/keystone/+bug/2030061>`_.
+
+  While the Keystone docs suggest that the ``member`` role should imply the
+  ``reader`` role, it has been seen at a customer that newly-generated app
+  creds in the Antelope release may need both the ``member`` and ``reader``
+  role specified.
+
+  Here are some SQL scripts you can call to first see if any app creds are
+  affected, and then add the reader role where needed. It is recommended to
+  backup the database before running these.
```

Comment:
nit: it would be nice to actually say how to back up the database here

Reply:
Done